docs/topics/auth/customizing.txt

```diff
@@ -1129,20 +1129,14 @@ code would be required in the app's ``admin.py`` file::
     class UserChangeForm(forms.ModelForm):
         """A form for updating users. Includes all the fields on
         the user, but replaces the password field with admin's
-        password hash display field.
+        disabled password hash display field.
         """
         password = ReadOnlyPasswordHashField()
 
         class Meta:
             model = MyUser
             fields = ('email', 'password', 'date_of_birth', 'is_active', 'is_admin')
 
-        def clean_password(self):
-            # Regardless of what the user provides, return the initial value.
-            # This is done here, rather than on the field, because the
-            # field does not have access to the initial value
-            return self.initial["password"]
-
```

Comment:
I would add `versionchanged` annotation, e.g.
```
.. versionchanged:: 3.2

    In older versions, ``ReadOnlyPasswordHashField`` is not
    :attr:`~django.forms.Field.disabled` by default and
    ``UserChangeForm.clean_password()`` is required to return the initial
    value, whatever the user provides.
```

Reply:
Ok. The `ReadOnlyPasswordHashField` itself is not documented with autodoc, so I included the `versionchanged` annotation in the section about the `UserChangeForm`.

Reply:
I think this note is the most important in the section about "Extending the existing User model", so I moved it there.